SpanishFilter/sections/antiadblock.txt

```diff
@@ -9,6 +9,10 @@
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/198826
+/^https:\/\/hunterscomics\.com\/wp-content\/themes(\/[a-f]{6}){2}\.js\?ver=/$script
```

Comment:
better with `domain`, the fewer common regular expressions, the better. But for popupular ads it would be ok.
```suggestion
/^https:\/\/hunterscomics\.com\/wp-content\/themes(\/[a-f]{6}){2}\.js\?ver=/$script,domain=hunterscomics.com
```